metis-normalization/src/main/java/eu/europeana/normalization/dates/edtf/InstantEdtfDate.java

```diff
@@ -158,6 +159,7 @@ public void removeTime() {
 
   public Integer getCentury() {
     final int century;
+
     // TODO: 25/07/2022 getEdtfDatePart() or getEdtfDatePart().getYear() might be null??
```

Comment:
Better to check both for nullity and if they are null we then throw an exception.